interfaces/apparmor/template.go

```diff
@@ -233,13 +233,15 @@ var defaultTemplate = []byte(`
 
   # Writable home area for this version.
   owner @{HOME}/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** wl,
+  owner @{HOME}/snap/@{SNAP_NAME}/shared/** wl,
```

Comment:
Reading about this, "common" makes sense from an internal code perspective for sure, I'm less sure from a developer perspective it would though.
"common" is implicit (as for "shared") that multiple entities can access to it at the same time, which isn't the case as @niemeyer suggested.

What you do actually end up with are versioned and version-independant areas. It's even more obvious when you ls in the data directory as you will have:
`version1/`, `version2/`, …, `current`. Why not then rather using `unversioned` which I guess will live better next to those versions and the current symlink? Thoughts?